### PR TITLE
add stubhub

### PR DESCRIPTION
**Pull request by setchy**
hey maintainers 👋 

first time contributing, and have a bunch more site lined up to share.  feedback welcomed

### CODE REVIEW

**Comment by kmpoppe** [comment]
Hey Adam, thanks for your contribution!

Is  https://en.m.wikipedia.org/wiki/File:Stubhub.svg also a valid logo for stubhub - it would be considerably more lightweight 😅

K

File: entries/s/stubhub.com.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "Ticketmaster": {
```
Comment (hkamran80) [inline entries/s/stubhub.com.json]:
```suggestion
  "StubHub": {
```

File: img/s/stubhub.com.svg [hunk 2]
```diff
@@ -0,0 +1 @@
... 7.747-.077m129.101-1.243c1.837.509 2 1.554 2.104 13.505.119 13.768.595 15.065 5.793 15.794 6.398.897 7.703-1.938 7.665-16.656-.034-12.69.093-13 5.304-13 3.411 0 3.531.458 3.533 13.48.001 11.136-.288 14.102-1.591 16.32-.388.66-.816 1.671-.951 2.246-.808 3.444-5.394 5.354-12.857 5.354-9.363 0-11.627-1.208-14.235-7.6l-1.307-3.2-.263-11.6c-.345-15.213.236-16.463 6.805-14.643m-84.113 7.817c-8.283 2.38-6.821 21.092 1.679 21.494 3.666.173 4.975-.127 6.591-1.509.873-.748 1.714-1.359 1.869-1.359 2.03 0 2.674-12.022.832-15.534-1.545-2.947-6.558-4.36-10.971-3.092m129.97.24c-1.068.406-1.982.794-2.03.862-2.816 3.976-2.785 3.887-2.95 8.735l-.156 4.611 1.435 2.448c3.776 6.443 13.939 5.972 15.862-.734.703-2.451.458-10.731-.37-12.536-1.708-3.719-6.951-5.224-11.791-3.386m-121.953 1.184c3.641 2.22 4.384 15.24.954 16.712-.582.25-1.54.848-2.128 1.328-1.371 1.119-4.529 1.436-5.93.595a62.09 62.09 0 0 0-2.285-1.284c-3.756-1.997-3.545-15.097.28-17.315 1.99-1.155 7.24-1.176 9.109-.036m131.164.634a38.625 38.625 0 0 0 1.633 1.891c.741.779 1.245 8.109.741 10.792-.741 3.954-7.493 7.242-10.558 5.142a55.207 55.207 0 0 0-2.147-1.385c-4.022-2.43-3.07-16.218 1.204-17.424 3.078-.868 8.065-.331 9.127.984"/></g></svg>
```
Comment (hkamran80) [inline img/s/stubhub.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 604 273.5"><path fill="#003168" d="M12.33 261.91c-3 1.2-4.7-.3-3.7-3.4l8.5-27.5a32 32 0 0 0 1.1-11.5L.03 46.11a5.7 5.7 0 0 1 5.3-6.3L598.53.01a5 5 0 0 1 5.4 5.5l-24.1 262.5a6.18 6.18 0 0 1-6.5 5.5l-504-32.4a29.72 29.72 0 0 0-11.4 1.9zm271.9-145c-6.7 0-12 2.1-15.9 6.1v-28.2h-12.6v58.8c0 8.6 2.6 15.7 7.7 21s12.1 8 20.9 8 15.9-2.7 21-8 7.7-12.4 7.7-21v-7.1c-.1-19.6-9.7-29.6-28.8-29.6zm0 12.3c5.3 0 9.3 1.3 12 4s4.1 7.2 4.1 13.4v7.1c0 5.2-1.4 9.2-4.3 12.3s-6.7 4.5-11.8 4.5-9-1.5-11.7-4.5-4.2-7.1-4.2-12.3v-7.2c0-6.2 1.4-10.7 4-13.4s6.7-3.9 11.9-3.9zm91.5 1.8h-39.9v-36.4h-13v87.3h13v-38.2h39.9v38.2h13.1v-87.3h-13.1v36.4zm111.9-14.1c-6.7 0-12 2.1-15.9 6.1v-28.2h-12.6v58.8c0 8.6 2.6 15.7 7.7 21s12.1 8 20.9 8 15.9-2.7 21-8 7.7-12.4 7.7-21v-7.1c-.1-19.6-9.8-29.6-28.8-29.6zm0 12.3c5.3 0 9.3 1.3 12 4s4.1 7.2 4.1 13.4v7.1c0 5.1-1.4 9.3-4.3 12.3s-6.8 4.5-11.8 4.5-9-1.5-11.7-4.5-4.2-7.1-4.2-12.3v-7.2c0-6.2 1.4-10.7 4-13.4s6.7-3.9 11.9-3.9zm-254.3 23.8c0 6.5-1 11.2-2.9 13.7s-5 3.7-9.7 3.7h-.4c-4.6 0-7.9-1.3-9.7-3.7-1.9-2.6-2.9-7.2-2.9-13.7v-34.4h-12.4v34.4c0 10.5 1.9 18.2 5.8 22.7 3.7 4.4 9.7 6.7 17.9 6.9h2.9c8.2-.2 14.2-2.5 17.9-6.9 3.8-4.6 5.8-12.2 5.8-22.7v-34.4h-12.3v34.4zm203.4 0c0 6.5-1 11.1-2.9 13.7-1.8 2.4-5 3.7-9.6 3.7h-.6c-4.6 0-7.9-1.3-9.7-3.7-1.9-2.6-2.9-7.2-2.9-13.7v-34.4h-12.4v34.4c0 10.5 1.9 18.2 5.8 22.7 3.7 4.4 9.7 6.7 17.9 6.9h3c8.2-.2 14.2-2.5 17.9-6.9s5.8-12.2 5.8-22.7v-34.4h-12.4v34.4h.1zm-303.6-18.4-30.3-10.1a12.44 12.44 0 0 1-4.8-2.8 4.85 4.85 0 0 1-1.7-3.4v-2c0-7.1 5.9-10.7 18-10.9h1.1c12.1 0 18.3 3.2 19 9.7l.1 1h13v-1.1c-.6-14.8-11.4-22.4-32.1-22.4h-1.3c-20.3.2-30.5 8.2-30.5 23.7v2a17.52 17.52 0 0 0 4 11.3c2.6 3.1 6.5 5.5 11.4 7.1l30 10.1c5.1 1.7 7.6 4.5 7.6 8.7v1.8c0 3.7-1.8 6.5-5.5 8.7s-9.2 3.4-16 3.4-12.1-1.1-15.8-3.4-5.3-5-5.3-8.7v-1.3h-12.9v1.3a21.7 21.7 0 0 0 9.1 18.2c6 4.4 14.3 6.7 24.9 6.7s19.2-2.2 25.1-6.7a21.45 21.45 0 0 0 9.1-18.2v-1.8a20.32 20.32 0 0 0-4.4-13.1 25.29 25.29 0 0 0-11.8-7.8zm43 31.5q-4.2-4.5-4.2-12.3v-22.4h13.5v-12.8h-13.5v-23.8h-12.6v59c0 8.6 2.6 15.7 7.7 21a26.55 26.55 0 0 0 17.2 7.8l1.2.1v-12.2l-.9-.1a14.48 14.48 0 0 1-8.4-4.3zm337.1-51h1.6v4.6h.8v-4.6h1.6v-.6h-4v.6zm7.3 3.1-1.4-3.7h-1.2v5.2h.7v-4.3l1.7 4.3h.5l1.6-4.3v4.3h.8v-5.2h-1.3l-1.4 3.7z"/></svg>
```

Here's a smaller SVG, as suggested by @kmpoppe

File: entries/s/stubhub.com.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    "domain": "stubhub.com",
+    "contact": {
+      "facebook": "Stubhub",
+      "twitter": "StubHub"
+    },
```
Comment (hkamran80) [inline entries/s/stubhub.com.json]:
```suggestion
```

Since they support 2FA, you don't need the `contact` object

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Comment by setchy** [comment]
thanks for the feedback @kmpoppe @hkamran80.  Pr updated :)

**Review by Carlgo11 — APPROVED** [review]

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**